queries.sql

```diff
@@ -92,4 +92,58 @@ SET weight_kg = weight_kg * -1;
 WHERE weight_kg < 0 ;
 SELECT * FROM animal;
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
```

Comment:
Please be sure to add the **owners** data into your data.sql file 🧐

```sql
INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);
```

Reply:
Also kindly add these to species table 

```sql
 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```

Reply:
I fixed it @mateo951 👍🏼